client/components/ProductOverview/AddToCart.jsx

```diff
@@ -1,24 +1,102 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
+import axios from 'axios';
 
 import '../../css/AddToCart.scss';
 
 const AddToCart = ({ currentStyle }) => {
   const [shirtSKU, setShirtSKU] = useState(null);
+  const [sizeChosen, setSizeChosen] = useState(null);
+  const [isAddToCart, setIsAddToCart] = useState(false);
+  const [showCart, setShowCart] = useState(false);
+  const [cartMessage, setCartMessaage] = useState(null);
+
+  const openSizeSelector = useRef(null);
 
   useEffect(() => {
     setShirtSKU(null);
```

Comment:
Can you explain what's going on here? Is the initial state being reset when the current style changes?

Reply:
Correct, this clears the cart message down below and allows for appropriate prompt messages to be displayed when user doesn't choose a shirt size or quantity.